.github/workflows/build.yml

```diff
@@ -188,10 +188,12 @@ jobs:
           vendor/bin/pcov clobber && \
           php -dpcov.enabled=1 -dpcov.directory=. -dpcov.exclude="~vendor~" vendor/bin/phpunit -c tests/phpunit.xml tests/PHPStan
 
-      - name: "Codecov.io"
+      - name: "Coveralls"
         env:
-          CODECOV_TOKEN: "${{ secrets.CODECOV_TOKEN }}"
-        run: "bash <(curl -s https://codecov.io/bash) -f tests/tmp/clover.xml"
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+           composer require twinh/php-coveralls --dev && \
+           vendor/bin/php-coveralls --verbose --coverage_clover=tests/tmp/clover.xml --json_path=tests/tmp/coveralls-upload.json
```

Comment:
Could you try following the custom GItHub Action guide instead: https://github.com/marketplace/actions/coveralls-github-action

Reply:
That GitHub Action is for LCOV formatted coverage only.
I've tried that, please see the first commits.

```
[error] "2020-08-27T02:10:55.888Z"  'error from lcovParse: ' 'Failed to parse string'
[error] "2020-08-27T02:10:55.889Z"  'input: ' '<?xml version="1.0" encoding="UTF-8"?>\n' +
```

https://github.com/phpstan/phpstan-src/runs/1034404107

Reply:
> There's an official github action repo by coveralls but it seems to be for js based projects only coverallsapp/github-action.

https://github.com/php-coveralls/php-coveralls/issues/273#issuecomment-533221094

Reply:
Yeah, that's why I wasn't successful previously. Thanks.